x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/risk_engine/risk_engine_privileges.ts

```diff
@@ -88,15 +120,18 @@ export const withRiskEnginePrivilegeCheck = <P, Q, B>(
     context: SecuritySolutionRequestHandlerContext,
     request: KibanaRequest<P, Q, B>,
     response: KibanaResponseFactory
-  ) => Promise<IKibanaResponse>
+  ) => Promise<IKibanaResponse>,
+  privilegeType: 'run' | 'enable' = 'enable'
```

Comment:
can we move this to the first argument so it isn't hidden after the route handler?